Vault 2.0 UI known issues (#2213)
hashicorp/web-unified-docs: content/vault/global/partials/important-changes/known-issue/root-token-kv-access-blocked-by-egp-policy.mdx

### Root token KV access blocked by EGP policy ((#root-token-kv-access-blocked-by-egp-policy)) <EnterpriseAlert inline="true" />

| Change      | Status | Vault edition | Affected version | Fixed version |
| ----------- | ------ | ------------- | ---------------- | ------------- |
| Known issue | Open   | Enterprise    | 2.0.0            | No

#### Issue

When a root token is used in the Vault UI to access a child namespace protected by

> an Endpoint Governing Policy (EGP), the request can be denied unexpectedly.

Reviewer: ```suggestion
If you use a root token in the Vault GUI to access a child namespace protected by
an Endpoint Governing Policy (EGP), Vault may deny the request unexpectedly.
```
Style correction: write in active voice, use "GUI" for the web UI

This affects UI flows that call `sys/internal/ui/mounts` with a namespace header.
In affected cases, the UI shows a `permission denied` error even though root
tokens are normally expected to bypass Sentinel policy checks. Equivalent CLI and

> API workflows are not affected.

Reviewer: ```suggestion
The unexpected rejection affects GUI flows that call `sys/internal/ui/mounts` with a namespace header.
In affected cases, the GUI shows a `permission denied` error even though root
tokens typically bypass Sentinel policy checks. Equivalent CLI and
API workflows are unaffected.
```
Style correction: write in active voice, avoid "this" as a pronoun (especially across setences)

#### Workaround

Use the CLI or API for the affected workflow.

If UI access is required, update the EGP policy logic to explicitly allow

> requests to `sys/internal/ui/mounts` for the affected namespace.

Reviewer: ```suggestion
If you require GUI access for an expected workflow, update the EGP policy logic to explicitly allow
requests to `sys/internal/ui/mounts` for the affected namespace.
```
